installation_wizard_datachoice

```diff
@@ -0,0 +1,87 @@
+When installing ownCloud Server & ownCloud Enterprise editions the administrator
+may choose one of 3 supported database products.
+
+SQLite
+^^^^^^
```

Comment:
Not sure if it makes a difference, but I seem to remember that we use "=" or "-" for undelines in other pages.